python/ray/tune/script_runner.py

```diff
@@ -84,6 +84,30 @@ def run(self):
             raise e
 
 
+def import_function(file_path, function_name):
```

Comment:
where does this code come from?

Reply:
Also, can you elaborate on why we need this function?

Reply:
This code comes from the script_runner init from before.

@ericl correct me if this is wrong - We want to be able to run an arbitrary script as a `Trial`, so we have to load the external file somehow, (the if cases are for python compatibility...)

Reply:
Yeah, this is needed if you want to run external code. The code is from stack overflow...